packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/focus-ring" as *;
+
+// TODO: Copy & pasted, clean up
```

Comment:
Still need to refine styles. These aren't yet following latest designs. Will update.